src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/ClientBaseTests.cs

```diff
@@ -174,6 +175,163 @@ public static void ClientBaseOfT_Sync_RoundTrip_Call_Using_NetTcpTransport()
         }
     }
 
+    [Fact]
```

Comment:
@StephenBonikowsky, can you comment on the suitability of putting the test in this part of the tree?